src/workerd/io/compatibility-date-test.c++

```diff
@@ -166,7 +167,7 @@ KJ_TEST("compatibility flag parsing") {
        "command line."_kj},
       CompatibilityDateValidation::CODE_VERSION);
   expectCompileCompatibilityFlags("2020-01-01", {"durable_object_alarms"_kj}, "(obsolete14 = true)",
-      {}, CompatibilityDateValidation::CODE_VERSION_EXPERIMENTAL);
+      {});
```

Comment:
Per the comment above, this test is intended to verify experimental flag enforcement. It now fails.

I think you should mark the `durable_object_alarms` with `$experimental`, so that this part of the test can be kept as it is. (You'll just need to change this line to pass `true` for `experimental.)

Otherwise, the test doesn't currently cover experimental validation at all. Note that, in general, you should always include a test for new functionality before submitting the PR for review.

Reply:
Fixed